tests/golden/rename-command/rename-command.bats

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bats
+
+load '../helpers/bats-support/load'
+load '../helpers/bats-assert/load'
+load '../helpers'
+
+@test "rename an entry" {
+  run coffer create /a
+  assert_output "[SUCCESS] Entry created at '/a'."
+
+  run coffer rename /a /b
+  assert_output "[SUCCESS] Renamed '/a' to '/b'."
+
+  run cleanOutput coffer view /
+  cat rename-an-entry.gold | assert_output -
+}
+
+@test "rename a directory" {
+  run coffer create /a/b/c
+  assert_output "[SUCCESS] Entry created at '/a/b/c'."
+
+  run coffer rename /a/b /d
+  assert_output "[SUCCESS] Renamed '/a/b/c' to '/d/c'."
+
+  run cleanOutput coffer view /
+  cat rename-a-directory.gold | assert_output -
+}
+
+@test "trying to rename without force" {
+  run coffer create /a/b/c
+  assert_output "[SUCCESS] Entry created at '/a/b/c'."
+
+  run coffer create /d/c
+  assert_output "[SUCCESS] Entry created at '/d/c'."
+
+  run coffer rename /a/b /d
+
+  assert_failure
+  cat rename-without-force.gold | assert_output --partial -
```

Comment:
The same comment [I made here](https://github.com/serokell/coffer/pull/32#discussion_r820249222) also applies to this test

Reply:
Done